### ISSUE TITLE
一个最后output 层包含deconvolution的网络，模型转换时出现output (Deconv2D): [[-1L, -1L, -1L, 1L]】？

### ISSUE THREAD

**Issue by akbarxie** [XiaoMi/mace#33]
最后一层为deconvolution 层：
Iv_p_038/deconv1/deconv1/bias_0_b2i (BufferToImage): []
output (Deconv2D): [[-1L, -1L, -1L, 1L]]
mace_output_node_output (ImageToBuffer): []
读取模型错误？？可能是什么原因

**Comment by TonyMou** [comment]
@akbarxie 可以提供一下这个deconvolution层的参数和输入输出shape吗？

**Comment by akbarxie** [comment]
tf.layers.conv2d_transpose(value, filters=1, kernel_size=3, strides=3, padding='same') 
不需要定义shape，底层会自动算shape。
mace是否只能识别tf.nn.conv2d_transpose这个操作

**Comment by akbarxie** [comment]
I have solved the problem by upgrade tensorflow from 1.6 to 1.8，Thank you.
